### PR TITLE
Update/2.25.0

### PR DESCRIPTION
**Pull request by k0gen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,71 @@
...
 
 ```
+start-cli auth login
+#Enter your StartOS password
 make install
 ```
 
-> **Tip:** You can also install the filebrowser.s9pk using **Sideload Service**
-> under the **Embassy > Settings** section.
+**Tip:** You can also install the filebrowser.s9pk using **Sideload Service** under the **StartOS > SETTINGS** section.
```
Comment (elvece) [inline README.md:65]:
```suggestion
**Tip:** You can also install the filebrowser.s9pk by sideloading it under the **StartOS > System > Sideload a Service** section.
```

File: manifest.yaml [hunk 2]
```diff
@@ -25,14 +30,17 @@ main:
...
-    success-message: Backend is reachable and responsive.
-    type: script
   web-ui:
     name: Web Interface
     success-message: Filebrowser is ready to be visited in your browser.
-    type: script
+    type: docker
+    image: main
+    system: false
+    entrypoint: /healthcheck.sh
```
Comment (elvece) [inline manifest.yaml:39]: Where does this file come from? I don't see it in this PR, is it from the upstream?

File: scripts/embassy.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
 export { properties } from "./procedures/properties.ts";
-export { health } from "./procedures/health.ts";
+//export { health } from "./procedures/health.ts";
```
Comment (elvece) [inline scripts/embassy.ts:2]: Remove comment

File: manifest.yaml (hunk 2 above)
Comment (k0gen) [reply manifest.yaml:39]: upstream all the way :)

**Review by elvece — APPROVED** [review]

File: scripts/embassy.ts (hunk 3 above)
Comment (k0gen) [reply scripts/embassy.ts]: done

**Pull request merged**